### PR TITLE
Support `postcss` language mode

### LINKED ISSUE #12: Provide option to define files to lint
I have my styles in `*.pcss` files. I want a way to tell `vscode-stylelint` that this files should be linted.

### PR DESCRIPTION
**Pull request by Guria**
resolves #12 by adding postcss document type support

### CODE REVIEW

**Comment by shinnn** [comment]
Currently there is no language support named "pcss", but "postcss". https://marketplace.visualstudio.com/items?itemName=ricard.PostCSS

**Comment by Guria** [comment]
Yeap, I can remove pcss then

**Comment by Guria** [comment]
updated

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "stylelint",
+  "name": "vscode-stylelint",
```
Comment (shinnn) [inline package.json]: Nope.

Comment (Guria) [inline package.json]: sorry, was an issue here when I have tried `npm i stylelint@latest` locally

Comment (Guria) [inline package.json]: fixed

**Pull request closed without merging**